sharing-data.html

```diff
@@ -261,105 +349,355 @@
-<p>Select the destination group from the options.</p>
-<p>Select an existing project and/or dataset as the destination.</p>
-<p>Click <span class="bold">OK</span> when done.</p>
-<p>If no destination is selected the images will be placed in <span class="bold">Orphaned</span> in the destination group.</p>
+
+<ul>
 
 <div class="warning">
 <p class="warning-title">Warning!</p>
 <p class="warning-text">There is a danger that some annotations such as tags and attachments on data being moved between groups could be lost if:
-<ul class="bullet-list">
-<li>the groups have different permission levels or </li>
-<li>the owners of the annotations are in the one group but not the other.</li>
+<ul>
+<p class="indent1">- the groups have different permission levels or </p>
+<p class="indent1">- the owners of the annotations are in the one group but not the other.</p>
 </ul>
 </p>
+
 </div><!-- End warning -->	
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p>Right-click on a Project, Dataset or one or more Images and select <span class="bold">Move to</span> from the menu.</p>
+<p>Select the destination group from the options.</p>
+<p>Select an existing Project and/or Dataset as the destination.</p>
+<p>Click <span class="bold">OK</span> when done.</p>
+<p>If no destination is selected the images will be placed in <span class="bold">Orphaned</span> in the destination group.</p>
+</ul>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.insight</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/sharingDataMoveInsight.jpg" class="align-center" src="images/sharingDataMoveInsight.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/sharingDataMoveWeb.jpg" class="align-center" src="images/sharingDataMoveWeb.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<h2><a class="anchor" id="default"></a>Changing your default group</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.insight</h3>
+
+<ul>
+
+<p>To change your default group, the group that's shown when you log in, go to the <span class="bold">Administration</span> tab.</p>
```

Comment:
For non-Admins, you change your own default group by (double?) clicking on yourself in the Projects tree (no Administration tab).